modules/geocoder_geofield/src/Plugin/Geocoder/Dumper/Geohash.php

```diff
@@ -27,7 +24,7 @@ class Geohash extends Geometry {
    * {@inheritdoc}
    */
   public function dump(Address $address) {
-    return parent::dump($address)->out('geohash');
+    return $this->geophp->load($this->getHandler()->dump($address), 'json')->out('geohash');
```

Comment:
Why? The parent returns the same. This the reason I made Geohash descendat of Geometry.